code/modules/clothing/outfits/standard.dm

```diff
@@ -178,6 +178,17 @@
 	mask = /obj/item/clothing/mask/cigarette/cigar/havana
 	l_pocket = /obj/item/lighter
 
+/// Used primarily by the outfit_sanity unit test, in-hand merging used to cause runtimes 3% of the time. See #66313 and #60901
+/datum/outfit/stacks_in_hands
```

Comment:
If this is unit test only put it behind test define, no need for it exist otherwise